configs/opa_bundles/default/authz.rego

```diff
@@ -43,9 +43,28 @@ allow if {
 allow if {
 	authn
 	input.kind == "usr"
+	input.action_type == "read"
 	input.user_id == input.resource_id
 }
 
+# Allow any user to invite other users as long as they
```

Comment:
I agree with this rule, but I think it's too weak: in general we don't want the inviter to set **any** user details (including `user_id`, `status`, `default_org_id`.

Also, a risk to mitigate: we need to ensure that any future changes in the user proto will be reflected here too.